content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -264,6 +264,102 @@ $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata
 $ kubectl exec -it $SOURCE_POD -c sleep curl http://httpbin.org/headers
 {{< /text >}}
 
+## Install Istio with access to all the external services by default
+
+An alternative to calling external services directly is to instruct Istio proxy to pass through the calls to all the
+external services. This option allows you to start evaluating Istio quickly, without controlling access to external
+services, and decide to [configure access to external services](#configuring-istio-external-services) later.
+
+Istio has an installation option that allows access to all external services for which no HTTP service exist inside
+the mesh and no HTTP/TCP `ServiceEntry` is defined. For example, if your mesh does not have an HTTP service on port 443,
+and you did not define a `ServiceEntry` on port 443, you can access any external service on port 443. Note, however,
+that once you create an HTTP service on port 443 or define any `ServiceEntry` for any host on
+port 443, all the access to port 443 will be blocked: Istio will fall back to the blocking-by-default behavior for that
```

Comment:
all the access -> all accesses
will be -> is

Reply:
fixed in https://github.com/istio/istio.io/pull/3140/commits/21cde1e825b85efa46c5b809d1ead2cdeccb6f28